internal/vds/vds.h

```diff
@@ -23,6 +23,12 @@ enum axis {
     SAMPLE    = 7,
 };
 
+enum coordinate_system {
+    IJ   = 0,
+    ILXL = 1,
+    CDP  = 2,
+};
+
```

Comment:
There is a coordinate system enum in cpp, which is used for slice. We should unify them. And also agree on coordinate system names. It's a different commit for sure, but it would make sense to genereralizing the existing one, and then make use of it in this commit

Reply:
Unified, but I feel like dividing it into commits (for internal renaming) is more pain then profit.
Not sure if we would like to change public one. But that's a different discussion.